src/apps/backend/modules/account/account-service.ts

```diff
@@ -19,12 +20,16 @@ export default class AccountService {
     password: string,
     username: string
   ): Promise<Account> {
-    return AccountWriter.createAccountByUsernameAndPassword(
+    const account = await AccountWriter.createAccountByUsernameAndPassword(
       firstName,
       lastName,
       password,
       username
     );
+    await NotificationService.createNotificationPreference({
+      accountId: account.id,
```

Comment:
Create an enum `NotificationChannel - Email, SMS, Push` and a helper - All.

add `notificationChannels: list[NotificationChannel]` as parameter `createNotificationPreference` method. `notificationChannels` should default to `All` in the argument.